examples/cron/compound-staking-cron/src/lib.rs

```diff
@@ -119,25 +119,25 @@ pub fn query(_deps: Deps, env: Env, msg: QueryMsg) -> StdResult<Binary> {
             plane: "COSMOS".to_string(),
             action: "COSMOS_INVOKE".to_string(),
             address: "".to_string(),
-            msg: to_json_vec(&claim_reward).unwrap(),
+            msg: to_json_vec(&claim_reward).unwrap().to_vec(),
         });
 
         // 3. compose cosmos msg to stake the claimed rewards
         let stake_reward = MsgDelegate {
             ty: "/cosmos.staking.v1beta1.MsgDelegate".to_string(),
             delegator_address: delegator_address.clone(),
             validator_address,
-            amount: vec![Coin {
-                denom: "usdt".to_string(),
+            amount: Coin {
+                denom: "lux".to_string(),
                 amount: reward_amount.into(),
-            }],
+            },
         };
 
         instructions.push(FISInstruction {
             plane: "COSMOS".to_string(),
             action: "COSMOS_INVOKE".to_string(),
             address: "".to_string(),
-            msg: to_json_vec(&stake_reward).unwrap(),
+            msg: to_json_vec(&stake_reward).unwrap().to_vec(),
```

Comment:
`msg: to_json_vec(&claim_reward).unwrap().to_vec()`, this can just be msg: `to_json_vec(&claim_reward).unwrap()`